### PR TITLE
Relationships done

### PR DESCRIPTION
**Pull request by qAziz060708**

### CODE REVIEW

File: Market/Market.DataAccess/Models/Categories.cs [hunk 1]
```diff
@@ -9,9 +9,14 @@ namespace Market.DataAccess.Models
     public class Categories
     {
         public int CategoryID { get; set; }
+
         public string CategoryName { get; set; }
+
         public string CategoryType { get; set; }
```
Comment (JmlKhon) [inline Market/Market.DataAccess/Models/Categories.cs:16]: remove empty space

File: Market/Market.DataAccess/Models/Customers.cs [hunk 2]
```diff
@@ -9,8 +9,22 @@ namespace Market.DataAccess.Models
...
         public int CustomerId { get; set; }
+
         public string Name { get; set; }
+
         public string ContactAdd { get; set; }
+
         public string Address { get; set; }
+
+
+        public List<Categories> categories { get; set; }
```
Comment (JmlKhon) [inline Market/Market.DataAccess/Models/Customers.cs]: Prop names start with Capital letter

File: Market/Market.DataAccess/Models/Deliveries.cs [hunk 3]
```diff
@@ -9,7 +9,11 @@ namespace Market.DataAccess.Models
     public class Deliveries
     {
        public int DeliveriesId { get; set; }
```
Comment (JmlKhon) [inline Market/Market.DataAccess/Models/Deliveries.cs]: DeliveryId. NOT deliveriesId

File: Market/Market.DataAccess/Models/Deliveries.cs [hunk 4]
```diff
@@ -9,7 +9,11 @@ namespace Market.DataAccess.Models
...
+
        public int CustomerId { get; set; }
+
        public DateTime Date { get; set; }
```
Comment (JmlKhon) [inline Market/Market.DataAccess/Models/Deliveries.cs]: aniqroq nomlash kerak! Date nima uchun? DeliveryDate ? CreatedDate

File: Market/Market.DataAccess/Models/Customers.cs [hunk 5]
```diff
@@ -9,8 +9,22 @@ namespace Market.DataAccess.Models
     public class Customers
```
Comment (JmlKhon) [inline Market/Market.DataAccess/Models/Customers.cs]: nima class nomi ko'plikda ? Customer qilamiz db kelishgandikku?

File: Market/Market.DataAccess/Models/Payment.cs [hunk 6]
```diff
@@ -9,7 +9,11 @@ namespace Market.DataAccess.Models
     public class Payment
     {
         public int PaymentId { get; set; }
+
         public int CategoryId { get; set; }
+
         public DateTime Date { get; set; }
```
Comment (JmlKhon) [inline Market/Market.DataAccess/Models/Payment.cs]: Change Date to proper name

**Review by JmlKhon — CHANGES_REQUESTED** [review]

File: Market/Market.DataAccess/Models/Customers.cs (hunk 5 above)
Comment (qAziz060708) [reply Market/Market.DataAccess/Models/Customers.cs]: u ozi digramma shunaqa

File: Market/Market.DataAccess/Models/Category.cs [hunk 7]
```diff
@@ -6,12 +6,17 @@
 
 namespace Market.DataAccess.Models
 {
-    public class Categories
+    public class Category
     {
         public int CategoryID { get; set; }
```
Comment (isfandiyoro5) [inline Market/Market.DataAccess/Models/Category.cs]: CategoryID emas CategoryId boladi

File: Market/Market.DataAccess/Models/ShoppingOrder.cs [hunk 8]
```diff
@@ -9,7 +9,14 @@ namespace Market.DataAccess.Models
     public class ShoppingOrder
     {
         public int OrderId { get; set; }
+
         public int CustomerId { get; set; }
+
         public DateTime Date { get; set; }
```
Comment (isfandiyoro5) [inline Market/Market.DataAccess/Models/ShoppingOrder.cs]: Shunda Date nima bolyabdi DeliveryDate yoki AcceptDate

File: Market/Market.DataAccess/Models/Seller.cs [hunk 9]
```diff
@@ -9,7 +9,12 @@ namespace Market.DataAccess.Models
     public class Seller
     {
         public int SellerId { get; set; }
+
         public int ProductId { get; set; }
+
         public string Name { get; set; }
```
Comment (isfandiyoro5) [inline Market/Market.DataAccess/Models/Seller.cs]: Shu joyida FirstName bn LastName qilish kkmasmi

File: Market/Market.DataAccess/Models/Customer.cs [hunk 10]
```diff
@@ -6,11 +6,25 @@
 
 namespace Market.DataAccess.Models
 {
-    public class Customers
+    public class Customer
     {
         public int CustomerId { get; set; }
+
         public string Name { get; set; }
```
Comment (isfandiyoro5) [inline Market/Market.DataAccess/Models/Customer.cs]: Shu joydayam FirstName LastName qilish kkmasmi

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

**Pull request merged**